org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/ScanProcess.java

```diff
@@ -221,32 +221,32 @@ private void validateRequest(ScanBean bean) throws ValidationException, Instanti
 	 * Checks the monitors in the scan request. This removes from the
 	 * collection of monitor names the name of any monitor that is a scannable in the scan.
 	 * Also, if the property {@code org.eclipse.scanning.server.useDefaultActivatedMonitors }
-	 * is <code>true</code>, and the monitor list is empty, the default monitors are added. 
+	 * is <code>true</code>, and the monitor list is empty, the default monitors are added.
 	 * @param gen point generator
 	 * @throws Exception
 	 */
 	private void checkMonitors(IPointGenerator<?> gen) throws Exception {
 		Collection<String> monitorNames = bean.getScanRequest().getMonitorNames();
-		
+
 		// We set any activated monitors in the request if none have been specified.
 		if (Boolean.getBoolean("org.eclipse.scanning.server.useDefaultActivatedMonitors")) {
 			if (bean.getScanRequest().getMonitorNames()==null) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)